app/models/vim_performance_state.rb

```diff
@@ -182,13 +182,12 @@ def self.capture_vm_disk_storage(obj, field)
     obj.send("#{field}_storage")
   end
 
-  def self.capture_numvcpus(obj)
-    return nil unless obj.kind_of?(VmOrTemplate) && obj.respond_to?(:hardware) && obj.hardware
-    # FIXME: this is a z-stream patch
-    # this method name should really be changed to #capture_logical_cpus to
-    # match the actual column being read, however, there are several reports
-    # depending on the name :numvcpus
-    # A larger patch should be done outside of a z-stream release
```

Comment:
Can we keep this comment.   There is also a corresponding comment somewhere in the metric.rb code that references this one and should be updated.  It's gnarly, but I don't want to lose this knowledge until we fix all the naming.

cc @blomquisg